prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -271,20 +277,38 @@ func getReviewers(rc reviewersClient, author string, files []github.PullRequestC
 			continue
 		}
 		leafReviewers = leafReviewers.Union(fileUnusedLeafs)
-		reviewers.Insert(popRandom(fileUnusedLeafs))
+		if useStatusAvailability {
+			if r := findNonBusyReviewer(ghc, log, busyReviewers, fileUnusedLeafs); r != "" {
```

Comment:
I'd be somewhat inclined to move all this logic inside the function, aka move useStatusAvailability into `findNonBusyReviewer()` and just always call this (returning popRandom(fileUnusedLeafs) when we're not interested in availability)